src/server/main_service.cc

```diff
@@ -1212,9 +1211,7 @@ facade::ConnectionContext* Service::CreateContext(util::FiberSocketBase* peer,
                                                   facade::Connection* owner) {
   ConnectionContext* res = new ConnectionContext{peer, owner};
 
-  if (owner->IsPrivileged() && !RequirePrivilegedAuth()) {
-    res->req_auth = false;
-  } else {
+  if (owner->IsPrivileged() && RequirePrivilegedAuth()) {
```

Comment:
we require auth only on the privileged port (admin port), because all the other ports are being taken care of by ACL's